backends-common/postgres/src/test/java/org/apache/james/backends/postgres/PostgresExtension.java

```diff
@@ -178,6 +178,10 @@ public void beforeEach(ExtensionContext extensionContext) {
     @Override
     public void afterEach(ExtensionContext extensionContext) {
         resetSchema();
+
+        if (!rlsEnabled) {
+            dropAllConnections();
+        }
```

Comment:
Another way without introducing more code: move `disposePostgresSession` from `afterAll` to `afterEach`?

Reply:
what disposePostgresSession close is connection of PostgresExtension, not connections created when running PostgresJamesServerMain